frontend/integration-tests/tests/kubevirt/utils/utils.ts

```diff
@@ -37,7 +35,7 @@ export function removeLeakableResource(leakedResources: Set<string>, resource) {
 }
 
 export function createResource(resource) {
-  execSync(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+  execSync(`echo '${JSON.stringify(resource)}' | kubectl apply -f -`);
```

Comment:
Is this a good idea to mix declarative and imperative approach in creating resources? Wouldn't it be safer to have a new method, i.e `applyResource`?

Reply:
Well, the purpose of this change is avoiding to create a new method. Of course it can has another method or maybe just use `execSync`.

And why do we need  `createResource` for one line code?